components/tabs/public/tab_interface.cc

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "components/tabs/public/tab_interface.h"
+
+namespace tabs {
+
+bool TabInterface::IsPartitionedTab() const {
+  return false;
+}
+
+std::optional<PartitionedTabVisualData>
+TabInterface::GetPartitionedTabVisualData() const {
+  return std::nullopt;
+}
```

Comment:
I see what you did here and I think this is interesting, but I'd rather have these default implementations in `brave/chromium_src` override for consistency as we have in other places where we add new functions to chromium classes.

Reply:
ah, I see now. Thanks for the details!

I'd say having `BraveTabInterface` would be a nice touch here, but if it will require a lot of changes in different places, then we might stick with `tab_interface.cc` as you currently have it implemented.

Another take here - declare our methods with `= 0` and implement them where required in Chromium (`tab_model.cc`, `tab_android.cc`). Maybe even declare them only on desktop (via buildflag guard) and implement only in `tab_model.cc` for now.